wikiupdate_api/sendhook.py

```diff
@@ -40,12 +40,13 @@ def sendhook():
     print(r.text)
 
 def main():
+    print("Hello from thread!")
     #schedule.every().day.at("12:00").do(sendhook)
```

Comment:
I guess that an additional thing you can do to lower the risk of the get request being sent before the server has started is to put a simple `sleep(10)` (or however long the server normally takes to start serving requests) here. There are no guarantees (the server could be slow) but it should give a first post after startup a bit faster than otherwise.


Probably not something we need in this project, just a though :)